docs/install_ovos_core.md

```diff
@@ -1,70 +1,185 @@
+
+**messagebus**
+
+`git clone https://github.com/OpenVoiceOS/ovos-messagebus`
+
+`pip install ~/ovos-messagebus`
+
+This is the nervous system of OVOS needed for modules to talk to eachother
+
+[Starting the Messagebus](starting_modules.md#messagebus)
+
+**listener**
+
+OVOS has updated thier listener to use `ovos-dinkum-listener` instead of `ovos-listener`.  It is code from [mycroft-dinkum](https://github.com/MycroftAI/mycroft-dinkum) adopted for use with the OVOS ecosystem.  Previous listeners are still available, but not recommended.
+
+`git clone https://github.com/OpenVoiceOS/ovos-dinkum-listener`
+
+`pip install ~/ovos-dinkum-listener`
+
+You now have what is needed for OVOS to use a microphone and its associated services.  WakeWords, HotWords, and STT
+
+[Starting the listener](starting_modules.md#listener)
+
+**PHAL**
+
+The OVOS Plugin based Hardware Abstraction Layer is what is used to allow the OVOS software to communicate with hardware devices such as the [OS system](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-system) or interacting with the [Mycroft Mark1 device](https://github.com/OpenVoiceOS/ovos-PHAL-plugin-mk1)
+
+The PHAL system consists of two interfaces.
+
+`ovos-phal` is the basic interface that normal plugins would use.
+
+`ovos-admin-phal` is used where superuser privleges are needed.
+
+**Be extreamly careful when installing `admin-phal` plugins as it gives full control over the host system**
+
+`git clone https://github.com/OpenVoiceOS/ovos-PHAL`
 
-### Developer launcher script
+`pip install ~/ovos-PHAL`
 
-`start-mycroft.sh` is available to perform common tasks.
+This just installs the basic system that allows the plugins to work.
 
+[Starting PHAL](starting_modules.md#phal)
 
-Assuming you installed ovos-core in your home directory, run:
+**audio**
 
-- `cd ~/ovos-core`
-- `./start-mycroft.sh debug`
+This is the service that is used by OVOS to play all of the audio.  It can be a voice response, or a stream from somewhere such as music, or a podcast.
 
-The "debug" command will start the background services (microphone listener, skill, messagebus, and audio subsystems) as
-well as bringing up a text-based Command Line Interface (CLI) you can use to interact with Mycroft and see the contents
-of the various logs. Alternatively you can run `./start-mycroft.sh all` to begin the services without the command line
-interface. Later you can bring up the CLI using `./start-mycroft.sh cli`.
+It also installs [OVOS Common Play](https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin) which can be used as a stand alone media player and is required for OVOS audio playback.
 
-The background services can be stopped as a group with:
+`git clone https://github.com/OpenVoiceOS/ovos-audio`
 
-- `./stop-mycroft.sh`
+`pip install ~/ovos-audio`
 
-### Automatically on boot
+This will enable a default TTS engine for voice feedback from your OVOS device.  There are plenty of options available here also.
```

Comment:
```suggestion
This will enable the default TTS (text-to-speech) engine for voice feedback from your OVOS device. However, plenty of alternative TTS engines are available.
```

I expect that solely one TTS engine is the default. Inform me if this is incorrect.

Reply:
by default the secondary fallback TTS is disabled, a user needs to set it explicitly

Reply:
@JarbasAl, ensure you tag me when responding to me, because I'm not subscribed to this issue. Otherwise, good to hear.